lib/private/Route/Router.php

```diff
@@ -82,7 +79,11 @@ public function __construct(
 	public function getRoutingFiles() {
 		if ($this->routingFiles === null) {
 			$this->routingFiles = [];
-			foreach (\OC_APP::getEnabledApps() as $app) {
+			$user = Server::get(IUserSession::class)->getUser();
+			$apps = $user === null
+				? $this->appManager->getEnabledApps()
+				: $this->appManager->getEnabledAppsForUser($user);
```

Comment:
```suggestion
			if ($user === null) {
				$apps = $this->appManager->getEnabledApps();
			} else {
				$apps = $this->appManager->getEnabledAppsForUser($user);
			}
```

I would like to suggest using a normal if-else statement here instead of the ternary operator for better readability.